### PR TITLE
Add LB forwarding rule

### PR DESCRIPTION
**Pull request by asarfaty**
And fix some lint & tests in the other LB rules
Still pending the vendor directory update

### CODE REVIEW

File: nsxt/resource_nsxt_lb_http_forwarding_rule_test.go [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    source_port = "7887"
+    inverse     = %s
+  }
+
+  http_reject_action {
+    reply_status  = "500"
+    reply_message = "rejected"
+  }
+
+  http_redirect_action {
```
Comment (annakhm) [inline nsxt/resource_nsxt_lb_http_forwarding_rule_test.go]: its weird that NSX allows to chain these together..

File: nsxt/resource_nsxt_lb_http_forwarding_rule.go [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"reply_status": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"reply_message": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
```
Comment (annakhm) [inline nsxt/resource_nsxt_lb_http_forwarding_rule.go]: I think this is required in NSX

File: nsxt/resource_nsxt_lb_http_forwarding_rule.go [hunk 3]
```diff
@@ -0,0 +1,496 @@
...
+				Description: "Redirect the request with a defined status and url",
+				Optional:    true,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"redirect_status": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"redirect_url": &schema.Schema{
```
Comment (annakhm) [inline nsxt/resource_nsxt_lb_http_forwarding_rule.go]: Same here - this is required

File: nsxt/resource_nsxt_lb_http_forwarding_rule_test.go (hunk 1 above)
Comment (asarfaty) [reply nsxt/resource_nsxt_lb_http_forwarding_rule_test.go]: Yeh...

File: nsxt/resource_nsxt_lb_http_forwarding_rule.go (hunk 2 above)
Comment (asarfaty) [reply nsxt/resource_nsxt_lb_http_forwarding_rule.go]: Done

File: nsxt/resource_nsxt_lb_http_forwarding_rule.go (hunk 3 above)
Comment (asarfaty) [reply nsxt/resource_nsxt_lb_http_forwarding_rule.go]: Done

File: website/docs/r/lb_http_forwarding_rule.html.markdown [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+  description = "lb_rule provisioned by Terraform"
+  display_name = "lb_rule"
+  match_strategy = "ANY"
+
+  tag = {
+    scope = "color"
+    tag   = "red"
+  }
+
+  body_condition {
```
Comment (annakhm) [inline website/docs/r/lb_http_forwarding_rule.html.markdown]: some formatting needed here

**Pull request merged**